src/resources/Pipelines/Conditions/ConditionInterfaces.ts

```diff
@@ -2,27 +2,103 @@ import {Paginated} from '../../BaseInterfaces';
 import {ListStatementSortBy} from '../../Enums';
 
 export interface ConditionModel {
+    /**
+     * The unique identifier of this statement.
+     */
     id: string;
+
+    /**
+     * The intented purpose of thi statement in an actual implementation.
+     */
     description: string;
+
+    /**
+     * The query pipeline language expression that defines this statement.
+     */
     definition: string;
+
+    /**
+     * The structured object of the query pipeline language expression of this statement.
+     */
     detailed: any;
+
+    /**
+     * @deprecated
+     * This property is exposed for backward compatibility reasons.
+     */
     childrenCount: number;
+
+    /**
+     * @deprecated
+     * This property is exposed for backward compatibility reasons.
+     */
     feature: string;
+
+    /**
+     * @deprecated
+     * This property is exposed for backward compatibility reasons.
+     */
     position: number;
+
+    /**
+     * @deprecated
+     * This property is exposed for backward compatibility reasons.
+     */
     ready: boolean;
+
+    /**
+     * @deprecated
+     * This property is exposed for backward compatibility reasons.
+     */
     parent?: string;
+
+    /**
+     * @deprecated
+     * This property is exposed for backward compatibility reasons.
+     */
     condition?: string;
 }
 
 export interface NewConditionModel {
+    /**
+     * The query pipeline language expression that defines this statement.
+     */
     definition: string;
+
+    /**
+     * The unique identifier of this statement.
+     */
     id?: string;
+
+    /**
+     * The intented purpose of this statement in an actual implementation.
+     */
     description?: string;
 }
 
 export interface ListConditionsOptions extends Paginated {
-    isOrderDescending?: boolean;
```

Comment:
Nice bonus fix ;)

Reply:
Do you think I should set this change as a breaking change? (we don't use it in the admin currently, but it's still an name change for a field)